packages/optimize/README.md

```diff
@@ -0,0 +1,348 @@
+
+# React Native AEP Optimize Extension
+
+[![npm version](https://badge.fury.io/js/%40adobe%2Freact-native-aepoptimize.svg)](https://www.npmjs.com/package/@adobe/react-native-aepoptimize)
+[![npm downloads](https://img.shields.io/npm/dm/@adobe/react-native-aepoptimize)](https://www.npmjs.com/package/@adobe/react-native-aepoptimize)
+
+
+`@adobe/react-native-aepoptimize` is a wrapper around the iOS and Android [Adobe Experience Platform Optimize Extension](https://aep-sdks.gitbook.io/docs/) to allow for integration with React Native applications.
+
+## Peer Dependencies
+
+The Adobe Experience Platform Optimize extension has the following peer dependency, which must be installed prior to installing the optimize extension:
+- [Core](../core/README.md)
+- [Edge](../edge/README.md)
+
+## Installation
+
+See [Requirements and Installation](https://github.com/adobe/aepsdk-react-native#requirements) instructions on the main page 
+
+Install the `@adobe/react-native-aepoptimize` package:
+
+NPM:
+```bash
+npm install @adobe/react-native-aepoptimize
+```
+
+Yarn:
+```bash
+yarn add @adobe/react-native-aepoptimize
+```
+
+Optimize native packages are not yet released and their github repo is private. Additional setup needs to be done in Podfile and build.gradle for integrating RN Optimize package in your RN application.
```

Comment:
Optimize is already released.  But do you want to include messaging maven snapshot info in the readme since user most likely use them together and might just look at one of the readme?